android/src/main/java/com/baseflow/permissionhandler/PermissionUtils.java

```diff
@@ -0,0 +1,31 @@
+package com.baseflow.permissionhandler;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.os.Build;
+
+import androidx.annotation.RequiresApi;
+import androidx.core.app.ActivityCompat;
+
+public class PermissionUtils {
+
+  @RequiresApi(api = Build.VERSION_CODES.M)
+   static boolean neverAskAgainSelected(final Activity activity, final String permission) {
```

Comment:
The line seems to start with one space to much (it doesn't align correctly with the rest of the method block):

```suggestion
  static boolean neverAskAgainSelected(final Activity activity, final String permission) {
```